salt/utils/minions.py

```diff
@@ -243,6 +243,8 @@ def _pki_minions(self):
             # Compiling pillar directly on the master, just return the master's
             # ID as that is the only one that is available.
             return [self.opts['id']]
+        if self.opts.get('__role') == 'minion': # Return self if running masterless
+            return [self.opts['id']]
```

Comment:
Seems like we could just combine this with the conditional above it?

Reply:
Thought about that but figured it might look too messy.

Reply:
In these cases, we usually just split it across multiple lines for readability.